rundeckapp/grails-app/utils/rundeck/codecs/JobsXMLCodec.groovy

```diff
@@ -107,6 +107,37 @@ class JobsXMLCodec {
         convertXMapToJobMap((Map) object)
     }
 
+    /**
+     * Checks if the given notification map follows or should follow the old format
+     */
+    static boolean useOldFormat(Map notificationMap, boolean isScheduleMap){
```

Comment:
What is this "isScheduleMap" flag for?
Please add some more javadoc for the params

Reply:
added